docs/development/intro.rst

```diff
@@ -112,23 +112,42 @@ Get set up
     Also, we shouldn't duplicate content from http://astropy.readthedocs.io/en/latest/#developer-documentation
     but link there instead.
 
-Commands to set up an environment for Gammapy development:
+The first steps are basically identical to https://cta-observatory.github.io/ctapipe/getting_started/index.html (until step 4, excluding 5)
+and http://astropy.readthedocs.io/en/latest/development/workflow/get_devel_version.html (up to *Create your own private workspace*).
+The following is a quick summary of commands to set up an environment for Gammapy development:
 
 .. code-block:: bash
 
+    # Fork the gammapy repository on GitHub, https://github.com/gammapy/gammapy
     cd code  # Go somewhere on your machine where you want to code
-    git clone https://github.com/gammapy/gammapy.git
+    git clone https://github.com/[your-github-username]/gammapy.git
     cd gammapy
     conda env create -f environment-dev.yml
     source activate gammapy-dev
+    # for conda versions >=4.4.0 you may have to execute
+    #'conda activate gammapy-dev' instead
+    git remote add gammapy git@github.com:gammapy/gammapy.git
```

Comment:
I would suggest to use the names `origin` and `upstream` for the remotes.
That's what most projects on Github use, and also [ctapipe](https://cta-observatory.github.io/ctapipe/getting_started/index.html#step-2-clone-your-forked-version-locally).
I see that Astropy has changed to other names, which is unfortunate IMO.

Reply:
Concerning names of remotes again: I see that the Astropy contributor docs used `astropy` for `upstream` and `YOUR_USERNAME` since over four years ago, and that there is a reasoning:
https://github.com/astropy/astropy/pull/1712#issuecomment-31387227
These names are clearer and what's used in https://hub.github.com/ which many people use when working with Github.

On the other hand, e.g. scikit-learn (http://scikit-learn.org/stable/developers/contributing.html) also uses `upstream` and `origin`, just like `ctapipe`.

@kosack - I guess you want to stick with `upstream` and `origin`?

So this is difficult what to recommend, and something people are always confused about.
In any case, I'd suggest showing `$ git remote -v` to show what the two remotes are when following our guidelines, and also to give people the command they need to find out what's what.

Reply:
@dcfidalgo - I'd suggest that you go ahead and finalise this PR, i.e. just keep the remote names as used by Astropy for now.

Reply:
Added a couple of sentences about the repo names, i think it should be sufficient to avoid confusion.